libpod/define/containerstate.go

```diff
@@ -141,11 +141,22 @@ type ContainerStats struct {
 	MemUsage      uint64
 	MemLimit      uint64
 	MemPerc       float64
-	NetInput      uint64
-	NetOutput     uint64
+	Network       map[string]ContainerNetworkStats
```

Comment:
Add a comment that the key is the interface name, not network name as some might expect.